### PR TITLE
update buildspec to output deployments.json

### PR DESCRIPTION
**Pull request by jesterhazy**
*Description of changes:*

update buildspec to output `deployments.json` file
this file will be used in our CD deployment pipeline.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: SUCCEEDED
* [Build Logs](https://3r4crpxqvk.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=2fd08e79-0c7e-4333-8ca1-db41ea331599%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by ChoiByungWook** [comment]
No EIA :(

**Comment by jesterhazy** [comment]
@ChoiByungWook not yet. i need some input from you and @ChuyangDeng on the right build commands. I'm hoping to get this merged in and then add those in another PR.

File: buildspec.yml [hunk 1]
```diff
@@ -25,22 +25,39 @@ phases:
...
+          ./scripts/publish.sh --version 1.12.0 --arch gpu
+        fi
+
+      # write deployment details to file
+      - |
+        if is-release-build; then
+          echo '[{
+          "repository": "sagemaker-tensorflow-serving",
+          "tags": [{
+            "source": "1.11.1-cpu",
```
Comment (ChoiByungWook) [inline buildspec.yml]: Just wondering, is there a reason we chose 1.11.1.

It looks like the training and the Python based TFS container both used 1.11.0

https://github.com/aws/sagemaker-tensorflow-container/tree/script-mode/docker/1.11.0
https://github.com/aws/sagemaker-tensorflow-container/tree/master/docker/1.11.0/final/py2

Comment (jesterhazy) [reply buildspec.yml]: 1.11.1 is the actual version of tensorflow/tensorflow serving in this container. we had a previous release with 1.11.0, but aren't maintaining images for that version anymore.

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: SUCCEEDED
* [Build Logs](https://3r4crpxqvk.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=d336dffa-0f8d-471a-91b8-038062230c7b%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Pull request merged**